Dockerfile

```diff
@@ -4,28 +4,86 @@ FROM continuumio/miniconda3
 # ENVIRONMENT #
 ###############
 ENV  TETHYS_HOME="/usr/lib/tethys" \
+     TETHYS_PERSIST="/var/lib/tethys_persist" \
+     TETHYS_APPS_ROOT="${TETHYS_HOME}/apps" \
      TETHYS_PORT=8000 \
-     TETHYS_PUBLIC_HOST="127.0.0.1" \
+     POSTGRES_PASSWORD="pass" \
+     TETHYS_DB_NAME='tethys_platform' \
      TETHYS_DB_USERNAME="tethys_default" \
      TETHYS_DB_PASSWORD="pass" \
      TETHYS_DB_HOST="db" \
      TETHYS_DB_PORT=5432 \
-     TETHYS_SUPER_USER="" \
-     TETHYS_SUPER_USER_EMAIL="" \
-     TETHYS_SUPER_USER_PASS=""
+     TETHYS_SUPER_USER="admin" \
+     TETHYS_SUPER_USER_PASS="pass" \
+     PORTAL_SUPERUSER_NAME="" \
```

Comment:
What is the difference between the `TETHYS_SUPER_USER` and the `PORTAL_SUPERUSER_NAME`? I think they should be the same thing.

Reply:
Tethys_super is for the database, and portal_super is the portal admin